test/db/formats/bflt

```diff
@@ -552,7 +552,7 @@ EXPECT=<<EOF
 ------ string (reloc target of above)
             ; UNKNOWN XREFS from fcn.0408aafc @ +0xc84, +0xc88, +0xd80, +0xd88
             ;-- str.Success:
-            0x040af7c4     .string "Success" ; len=7                   ; RELOC TARGET 32 
+            0x040af7c4     .string "Success" ; len=8                   ; RELOC TARGET 32 
```

Comment:
This looks wrong.

Reply:
no is correct. that is not the length but the size in bytes of the string